imdb_django/models.py

```diff
@@ -0,0 +1,134 @@
+# Create your models here.
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        app_label = 'genre_app'
+
+    def __str__(self):
+        return self.name
+
+
+class Title(models.Model):
```

Comment:
The best place for all the models is an app `imdb_dajngo` or you can create a new app. However, models should not be in `management/commands`

Reply:
The models file is in the app.